TaschenrechnerUnitTests/InformationTechnology/NumberSystemConverter.cs

```diff
@@ -0,0 +1,54 @@
+﻿using NUnit.Framework;
+using System.Linq;
+using Taschenrechner.Classes;
+
+namespace TaschenrechnerUnitTests
+{
+    class NumberSystemConverter
+    {
+        [Test]
+        public void Convert_110FromBinaryToDecimal_Return6()
```

Comment:
bennenung ist nicht optimal bei den test funktionen.
vorschlag: ,,ConvertFromBinaryToDecimal"
würde zumindest keine zahlen im namen mit einbinden.

wäre auch schön wenn positives und false cases mit in der bennenung beachtet werden.